File: src/main/java/com/fwdekker/randomness/integer/IntegerSettingsAction.java
package com.fwdekker.randomness.integer;

import com.fwdekker.randomness.SettingsAction;
import com.fwdekker.randomness.SettingsDialog;


/**
 * Controller for random integer generation settings.
 */
public final class IntegerSettingsAction extends SettingsAction {
    @Override
    public SettingsDialog createDialog() {
        return new IntegerSettingsDialog();
    }

    @Override
    public String getTitle() {
        return "Integer Settings";
    }
}

File: src/main/kotlin/com/fwdekker/randomness/integer/IntegerSettingsAction.kt
package com.fwdekker.randomness.integer

import com.fwdekker.randomness.SettingsAction


/**
 * Controller for random integer generation settings.
 */
class IntegerSettingsAction : SettingsAction() {
    public override fun createDialog() = IntegerSettingsDialog()

    public override fun getTitle() = "Integer Settings"
}
